crypto/keys/bcrypt/base64.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2011 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package bcrypt
```

Comment:
Confirmed this file is identical to https://cs.opensource.google/go/x/crypto/+/master:bcrypt/base64.go.